File: 2873-maximum-value-of-an-ordered-triplet-i/2873-maximum-value-of-an-ordered-triplet-i.java
class Solution {
    public long maximumTripletValue(int[] nums) {
        long ans=Integer.MIN_VALUE;;
        int n=nums.length;
        for(int i=0;i<n;i++)
        {
            for(int j=i+1;j<n;j++)
            {
                for(int k=j+1;k<n;k++)
                {
                    ans=(long)Math.max(ans,(long)(nums[i]-nums[j])*nums[k]);
                }
            }
        }
        if(ans<0)
            ans=0;
        return ans;
    }
}

File: 2873-maximum-value-of-an-ordered-triplet-i/2873-maximum-value-of-an-ordered-triplet-i.py
class Solution:
    def maximumTripletValue(self, nums: List[int]) -> int:
        ans=-999999;
        for i in range(len(nums)):
            for j in range(i+1,len(nums)):
                for k in range(j+1,len(nums)):
                    ans=max(ans,(nums[i]-nums[j])*nums[k])
        return 0 if ans<0 else ans